treeherder/etl/perf_data_adapters.py

```diff
@@ -90,16 +90,21 @@ def _round(num):
         return round(num, 2)
 
     @staticmethod
-    def _calculate_summary_data(job_id, result_set_id, push_timestamp, results):
+    def _extract_summary_data(suite_data, summary):
+        suite_data["geomean"] = summary["suite"]
+        return suite_data
+
+    @staticmethod
+    def _calculate_summary_data(job_id, result_set_id, push_timestamp,
```

Comment:
It would be nice to avoid unnecessary changes here. I think you can do that just by pushing `results` up to the previous line and reverting the geomean changes below.